config/locales/en/course/assessment/assessments.yml

```diff
@@ -17,7 +17,7 @@ en:
         assessment:
           autograded_assessment: 'Autograded assessment'
           attempt: 'Attempt'
-          condition_not_satisfied: 'You do not fulfill the requirements of the assessment'
+          condition_not_satisfied: 'You do not fulfill the requirements of the assessment, or the assessment is not open yet'
```

Comment:
This is abiguous for the user, which is it?

Reply:
I think you can keep the original title there since we are going to show the details of the requirements anyway